cmd/helm/installer/uninstall.go

```diff
@@ -21,13 +21,12 @@ import (
 
 	"github.com/ghodss/yaml"
 
+	"k8s.io/helm/pkg/kube"
 	"k8s.io/kubernetes/pkg/api"
 	kerrors "k8s.io/kubernetes/pkg/api/errors"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
 	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
-
-	"k8s.io/helm/pkg/kube"
```

Comment:
Never heard of this best practice. But if project follows unwritten best practices regarding style, I'm happy to oblige. Just thought I was done with these, when I switched to writing GO. :)